project2/index.html

```diff
@@ -6,28 +6,56 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:700,900" rel="stylesheet">
```

Comment:
If you bring both weights (`700` and `900`) into your project, remember to use both (or otherwise update this to only call the one font you need).